modules/ensemble/lib/widget/confirmation_input.dart

```diff
@@ -34,8 +36,8 @@ class ConfirmationInput extends StatefulWidget
       'fieldType': (input) =>
           _controller.fieldType = Utils.optionalString(input),
       'inputType': (type) => _controller.inputType = Utils.optionalString(type),
-      'obscureText': (type) => _controller.obscureText = Utils.optionalString(type),
-      'obscureSymbol': (typeCustom) => _controller.obscureSymbol = Utils.optionalString(typeCustom),
+      'obscureText': (type) => _controller.obscureText = Utils.optionalBool(type),
```

Comment:
@TheNoumanDev looks like you just ignored the backward compatability comment that I had posted. How is this backward compatible!

Reply:
sorry no need for backward compatability and obscureText is a new property anyway that just went in recently